packages/common/src/initiatives/_internal/InitiativeUiSchemaEdit.ts

```diff
@@ -187,51 +161,154 @@ export const buildUiSchema = async (
-            scope: "/properties/status",
-            type: "Control",
-            labelKey: `${i18nScope}.fields.status.label`,
+            type: "Section",
+            labelKey: "shared.sections.heroBannerAppearance.label",
             options: {
-              control: "hub-field-input-select",
-              enum: {
-                i18nScope: `${i18nScope}.fields.status.enum`,
+              section: "block",
+              helperText: {
+                labelKey: "shared.sections.heroBannerAppearance.helperText",
               },
             },
+            elements: [
+              {
+                labelKey: `${i18nScope}.fields.hero.label`,
+                scope: "/properties/view/properties/hero",
+                type: "Control",
+                options: {
+                  control: "hub-field-input-tile-select",
+                  layout: "horizontal",
+                  labels: [
+                    `{{${i18nScope}.fields.hero.map.label:translate}}`,
+                    `{{${i18nScope}.fields.hero.image.label:translate}}`,
+                  ],
+                  descriptions: [
+                    `{{${i18nScope}.fields.hero.map.description:translate}}`,
+                    `{{${i18nScope}.fields.hero.image.description:translate}}`,
+                  ],
+                  icons: ["map-pin", "image"],
+                },
+              },
+              {
+                labelKey: `${i18nScope}.fields.featuredImage.label`,
+                scope: "/properties/view/properties/featuredImage",
+                type: "Control",
+                rule: {
+                  effect: UiSchemaRuleEffects.HIDE,
+                  condition: {
+                    scope: "/properties/view/properties/hero",
+                    schema: { const: "map" },
+                  },
+                },
+                options: {
+                  control: "hub-field-input-image-picker",
+                  imgSrc: getAuthedImageUrl(
+                    options.view?.featuredImageUrl,
+                    context.requestOptions
+                  ),
+                  maxWidth: 727,
+                  maxHeight: 484,
+                  aspectRatio: 1.5,
+                  sizeDescription: {
+                    labelKey: `${i18nScope}.fields.featuredImage.sizeDescription`,
+                  },
+                },
+              },
+              {
+                labelKey: `${i18nScope}.fields.featuredImage.altText.label`,
+                scope: "/properties/view/properties/featuredImageAltText",
+                type: "Control",
+                rule: {
+                  effect: UiSchemaRuleEffects.HIDE,
+                  condition: {
+                    scope: "/properties/view/properties/hero",
+                    schema: { const: "map" },
+                  },
+                },
+              },
+            ],
           },
+          // NOTE: this will round trip but as far as i can tell it is not displayed on the view
```

Comment:
Yeah, technically any item-backed entity can have hero actions that round trip. If they are using the standard hero in the shared entity view, they should render in the hero, but since we're doing a fancy custom hero for initiatives, they don't render anywhere on that view.